packages/superadmin/components/products/createProduct.tsx

```diff
@@ -1,17 +1,47 @@
-import { Formik, Form } from "formik";
-import { productSchema } from "./schema/productSchema";
+import { Formik, Form } from 'formik';
+import { toast } from 'react-toastify';
+import { useRouter } from 'next/router';
+import { NextComponentType } from 'next';
+import { useEffect, useState } from 'react';
 
-import ProductInfoForm from "./forms/productInfoForm";
-import PhotosForm from "./forms/photosForm";
-import MetaForm from "./forms/metaForm";
-import { userAPI } from "../../APIs";
+import { userAPI } from 'APIs';
+import { productSchema } from '@/components/products/schema/productSchema/index';
+import MetaForm from '@/components/products/forms/metaForm';
+import PhotosForm from '@/components/products/forms/photosForm';
+import CategoryForm from '@/components/products/forms/categoryForm';
+import ProductInfoForm from '@/components/products/forms/productInfoForm';
 
-import { useRouter } from "next/router";
-import CategoryForm from "./forms/categoryForm";
-import { useEffect, useState } from "react";
-import { toast } from "react-toastify";
-
-const CreateProduct = () => {
+interface FormDataInterFace {
```

Comment:
make models folder and place this interface on it

Reply:
Done